### PR TITLE
DM-3021: includes (dm-2997) bug/code fixes

### PR DESCRIPTION
**Pull request by bradjohnson92008**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-3021
https://agile6.atlassian.net/browse/DM-2997

## Description - what does this code do?
1.  Removes external link icon in Vet crisis modal
2.  Updates the chevron \/ color in the "Browse by locations" drop down in the header.
3.  Search filter area now displays Categories on top of VISN/Facility filters.

## Testing done - how did you test it/steps on how can another person can test it 
1.  Navigate to '/' home page.
2. Click on the "Talk to the Veterans Crisis line now".  
  => ensure that the external icon is removed from links in the modal.
image

3.  Navigate to /search.
4. Open the Filters accordion.
  => ensure that the Categories are displayed at the top of the filters area and the "Originating and Adopting" location drop downs are displayed at the bottom... and that the filters still work correctly.
image

5.  View the "Browse by Locations" drop down in the header.
  => ensure that the drop down arrow color matches the text... and that the drop down hover, and expanded view works correctly.
image

## Acceptance criteria
1.  Remove external link icon in Vet crisis modal
2. Get color right in drop down Browse by location (header dropdown)
3. Updates search filters area so that Categories are on top of the "Originating and Adopting" location drop downs

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/views/shared/_veterans_crisis_line_modal.html.erb [hunk 1]
```diff
@@ -38,9 +38,6 @@
               <p>Call TTY if you have hearing loss&nbsp;<a href="tel:+18007994889" aria-label="8 0 0. 7 9 9. 4 8 8 9." class="usa-link usa-link--external">800-799-4889</a></p>
             </li>
           </ul>
-          <p>
```
Comment (emyl3) [inline app/views/shared/_veterans_crisis_line_modal.html.erb:41]: I thought when Becca went over this she just meant to get rid of this icon on the phone numbers or am I misremembering?
Screen Shot 2021-12-02 at 6 08 38 PM

File: app/assets/stylesheets/dm/_classes.scss [hunk 2]
```diff
@@ -338,3 +338,19 @@ to confirm the class DOES NOT EXIST before adding a new class
 .maxw-none {
   @include u-maxw('none', !important);
 }
+
+.usa-nav__primary button[aria-expanded="false"], .usa-nav__primary button[aria-expanded="true"]{
```
Comment (emyl3) [inline app/assets/stylesheets/dm/_classes.scss:342]: the chevron looks place slightly higher than in the figma -- maybe by a couple pixels (not sure if that's just me)

**Review by emyl3 — COMMENTED** [review]
If becca already approved the design changes for this then please ignore my comments. If not, I think once I'm done with trying to deploy the category search speed stuff we can put this on dev for Becca to review :) Good work, Brad!

File: app/views/shared/_veterans_crisis_line_modal.html.erb (hunk 1 above)
Comment (bradjohnson92008) [reply app/views/shared/_veterans_crisis_line_modal.html.erb:41]: Your correct... I've updated the code and PR.  Thx Elisa.. Yes - I'll wait til the category speech stuff is merged back in to Master... and get that merged in to this.. before merging this back to Master.

File: app/assets/stylesheets/dm/_classes.scss (hunk 2 above)
Comment (bradjohnson92008) [reply app/assets/stylesheets/dm/_classes.scss:342]: I know.. Josh and I went back and forth on this.. a few times..  I did show this to Becca and she said it was good... but as you mentioned.. after this is merged back in to Master I'll deploy this to DEV and have Becca give it a final OK.

**Review by JoshingYou1 — APPROVED** [review]
Everything looked good to me Brad. Nice work!!!

**Pull request merged**